src/components/structures/LoggedInView.tsx

```diff
@@ -484,6 +469,25 @@ class LoggedInView extends React.PureComponent<IProps, IState> {
                 }
                 break;
 
+            case Key.DIGIT_0:
+            case Key.DIGIT_1:
+            case Key.DIGIT_2:
+            case Key.DIGIT_3:
+            case Key.DIGIT_4:
+            case Key.DIGIT_5:
+            case Key.DIGIT_6:
+            case Key.DIGIT_7:
+            case Key.DIGIT_8:
+            case Key.DIGIT_9:
```

Comment:
if we were using a completely different language we could switch on the type rather than have whatever this can be described as.

Reply:
equally switch ranges would be nice